project-base/storefront/components/Basic/NotImplementedYet/NotImplementedYet.tsx

```diff
@@ -0,0 +1,77 @@
+import { SyntheticEvent, useState } from 'react';
+import { twJoin } from 'tailwind-merge';
+import { showErrorMessage } from 'utils/toasts/showErrorMessage';
+import { twMergeCustom } from 'utils/twMerge';
+
+const notImplementedMessage = 'Not implemented yet';
+
+export const notImplementedYetHandler = (e: SyntheticEvent): void => {
+    showErrorMessage(notImplementedMessage);
+    e.preventDefault();
+};
+
+const notImplementedTagTwClass =
+    'whitespace-nowrap rounded-sm bg-red p-1 text-center text-xs font-normal text-white opacity-100';
+const notImplementedTagPositionedTwClass =
+    'whitespace-nowrap rounded-sm bg-red p-1 text-center text-xs font-normal text-white opacity-100 absolute top-0 left-1/2 -translate-x-1/2 -translate-y-1/2';
+const notImplementedBorderTwClass = 'border border-dashed border-red';
+
+export const NotImplementedYetWrapper: FC = ({ children }) => {
+    return (
+        <div className={twJoin('relative', notImplementedBorderTwClass)}>
+            {children}
+            <div className={notImplementedTagPositionedTwClass}>{notImplementedMessage}</div>
+        </div>
+    );
+};
+
+export const NotImplementedYetInject: FC = () => {
+    return (
+        <div
+            className={twJoin(
+                'pointer-events-none absolute bottom-0 left-0 right-0 top-0',
+                notImplementedBorderTwClass,
+            )}
+        >
+            <div className={twMergeCustom(notImplementedTagPositionedTwClass, 'top-0 -translate-y-0')}>
+                {notImplementedMessage}
+            </div>
+        </div>
+    );
+};
```

Comment:
`NotImplementedYetInject` works only if the parent has a position other than `position: static;`. I don't think this should be a huge issue since it is demonstrated on the `Styleguide` page, but I wonder if there might be an easy solution.